serveradmin/powerdns/__init__.py

```diff
@@ -0,0 +1,18 @@
+class PowerdnsRouter:
```

Comment:
We want the default behaviour here which should be given if not specified as far as I understand but I can make it explicit.

```
If no router has an opinion (i.e. all routers return None), only relations within the same database are allowed.
```

Reply:
I am not that familiar with Routers, so I had to read it too. I guess it is more about this statement:
```
A router doesn’t have to provide all these methods – it may omit one or more of them. If one of the methods is omitted, Django will skip that router when performing the relevant check.
```
Which is fine. Question was more about that I am not that familiar with Routers.